pom.xml

```diff
@@ -5,16 +5,16 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>5.9</version>
+    <version>5.17</version>
     <relativePath />
   </parent>
 
   <properties>
-    <java.version>17</java.version>
-    <jenkins.version>2.502</jenkins.version>
+    <!-- https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/ -->
+    <jenkins.baseline>2.479</jenkins.baseline>
+    <jenkins.version>${jenkins.baseline}.3</jenkins.version>
```

Comment:
It is recommended to use the lastest minor in an LTS line. Right now, `2.479.3` is recommended as a baseline https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/